### ISSUE TITLE
Usage behind reverse proxy - allow change of set_real_ip_from

### ISSUE THREAD

**Issue by dhemeier** [crazy-max/docker-matomo#8]
Hey @crazy-max,

many thanks for your implementation (Issue #7) with the new Geo IP 2 database. Great work!

My last issue with the GeoIP is the usage with a reverse proxy configuration (inside a Kubernetes cluster on AWS). The current default `set_real_ip_from` implementation use `0.0.0.0/32` as trusted sources. My internal cluster IP is out of this range and this results in a not working RealIP-Module (http://nginx.org/en/docs/http/ngx_http_realip_module.html).

So now I need a way to change this value. In my case I could just use `set_real_ip_from 0.0.0.0/0;`
because I know, that the traffic can only access this container by the reverse proxy. But in general you can't use this as default (Attackers can then simply change the ip to whatever they want, if they can access the container directly and set a `x-forwarded-for` header).

I think a possible solution is to add a new ENV-Variable and assign them to the Nginx config in the entrypoint.sh?

Many thanks in advance
Dennis

**Comment by crazy-max** [comment]
Hey @dhemeier, yes we can use an env var for this you right. Btw your EC2 instances are behind ELB ? If so you should check the VPC CIDR on Amazon console.

**Comment by dhemeier** [comment]
Hey @crazy-max,

to be exactly, its not the EC2-ELB, the current IP comes from the Kubernetes Ingress-NGINX - ELB -> nginx-ingress -> Matomo ;-) But I could use the internal cluster CIDR, yes.

**Comment by dhemeier** [comment]
@crazy-max yep, integration now fully working (GeoIP2 with Location, City, Region, Continent, ...). Hope you continue do your great work, maybe the donate helps a little bit 👍

**Comment by crazy-max** [comment]
@dhemeier Thanks a lot for your support !
And it would be nice if you can create a PR with a basic K8s example :slightly_smiling_face:
